contrib/coreos/user-data.example

```diff
@@ -159,3 +159,8 @@ write_files:
     content: |
       [Coredump]
       Storage=none
+  - path: /etc/systemd/system/ntpd.service.d/debug.conf
+    content: |
+      [Service]
+      ExecStart=
+      ExecStart=/usr/sbin/ntpd -g -n -f /var/lib/ntp/ntp.drift
```

Comment:
This is only going to work for HVM by default. To work with PV, you'll need to "sysctl -w xen.independent_wallclock=1" in order to allow the PV clock to be settable independently from the underlying Xen host (which is already running NTP, providing the wallclock that the PV sees). PV maps the time to a hypervisor call to ask the underlying host.  HVM, and raw-iron linux clocks, keep track of the time using linux kernel jiffy counters, which can drift wildly when cycles are stolen by other VMs on the same Xen host machine.